pkg/operator/sync.go

```diff
@@ -329,6 +379,22 @@ func (optr *Operator) syncMachineConfigPools(config *renderConfig) error {
 		}
 	}
 
+	userdatas := []string{
+		"manifests/master.userdata_secret.yaml",
+		"manifests/worker.userdata_secret.yaml",
```

Comment:
we'll need a watcher on those secrets so if they get changed manually, the MCO can reconcile them (much like what we do for pools or any other resource the MCO manages)